api-controllers/routes/admin.js

```diff
@@ -32,6 +32,12 @@ router.route('/backups')
 router.route('/backups/:backup_guid/delete')
   .post(controller.handler('deleteBackup'))
   .all(commonMiddleware.methodNotAllowed(['POST']));
+router.route('/config/create')
+  .post(controller.handler('createUpdateConfig'))
```

Comment:
change the HTTPMethod to PUT. POST is generally for create and PUT for update. In this case since updates will keep happening its better to make this PUT.